py/confluent/build/download_artifacts/global_config.py

```diff
@@ -0,0 +1,16 @@
+class DownloadArtifactsConfig:
```

Comment:
Add documentation, what is the purpose of this struct. What do you intend to do with this.